handlers/aws/utils.py

```diff
@@ -123,7 +123,7 @@ def wrapper(lambda_event: dict[str, Any], lambda_context: context_.Context) -> s
         # it should not prevent all other events to be ingested.
         except Exception as e:
             if apm_client:
-                apm_client.capture_exception()
+                apm_client.capture_exception()  # type: ignore
```

Comment:
After upgrading to elastic-apm 6.23.0, mypy started complaining about missing type hints. 

The same complaints didn't happen with elastic-apm 6.19.0, and I guess this changed starting from 6.19.0+ version due to https://github.com/elastic/apm-agent-python/pull/1925

Reply:
Since elastic-apm doesn't have type hints, I'm going to add `type: ignore` for now.